filer/management/commands/import_files.py

```diff
@@ -81,7 +80,7 @@ def walker(self, path=None, base_folder=None):
         This method walk a directory structure and create the
         Folders and Files as they appear.
         """
-        path = path or self.path
+        path = path or self.path if self.path is not False else ''
```

Comment:
path must be a string in any case, I think we should rewrite this as `path or self.path or ''`

Reply:
ok - will update it now.